index.html

```diff
@@ -0,0 +1,231 @@
+<!Doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form </title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
```

Comment:
Please use section tag instead of a div because the section is semantic tag .

Reply:
- [ ] Good suggestion! But please double-check your message for any typos. This step will help ensure a clear understanding of your suggestion. Additionally, it would be beneficial to provide a more detailed explanation of why using a section is preferable to using a div.

   ```
     Example: 
     Please consider using the <section> tag instead of a <div> tag, as the <section> tag is a semantic element.
   ```
   
  #### Remember that you can edit your review comments here:
  ![image](https://user-images.githubusercontent.com/107952724/235543841-6988b256-c4cb-4d52-b4d3-fd54c6d2adfc.png)